src/index.html

```diff
@@ -9,10 +9,66 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="iMac image"
+      />
+
+      <p class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__stars">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviews">Reviews: 5</div>
```

Comment:
You need to use the semantic tag for text instead of div, fix it everywhere
```suggestion
        <p class="card__reviews">Reviews: 5</p>
```